src/main/kotlin/graphics/scenery/DetachedHeadCamera.kt

```diff
@@ -1,13 +1,13 @@
 package graphics.scenery
 
-import org.joml.Matrix4f
-import org.joml.Vector3f
 import com.jogamp.opengl.math.Quaternion
```

Comment:
This import should be removed.

Reply:
Then we should open an issue for that. It was not added in this PR.